verifiers/near.js

```diff
@@ -16,6 +16,17 @@ export default class NearVerifier {
           })
         return contractsObject.length
     }
+    async getAccountAge(accountId) {
+        const api = new NearApi();
+        const result = await api.getAccountInfo(accountId);
+        return result.account[0]?.created?.block_timestamp
+    }
+    async getAccountBalance(accountId) {
```

Comment:
The same thing with this method, it should be responsibility of NearApi class to return the necessary data, as the verifier class should only provide verifications to any kind of information that you are trying to trust.